lib/services/RenderingControl.js

```diff
@@ -111,13 +111,13 @@ class RenderingControl extends Service {
   }
 
   /**
-   * (Un)set bass of a speaker.
-   * @param {boolean} loudness Should it be with or without bass?
+   * Set bass of a speaker.
+   * @param {boolean} bass desired level of bass
    */
   async SetBass (bass) {
     return this._request('SetBass', {
       InstanceID: 0,
-      DesiredBass: (bass ? '1' : '0')
```

Comment:
Are you sure about this? Maybe explain in the comments what the min/max value is.